tensorflow/python/ops/image_ops_test.py

```diff
@@ -4071,6 +4071,14 @@ def testShape(self):
         self.assertEqual(image.get_shape().as_list(),
                          [None, None, channels or None])
 
+  def testEmptyTensor(self):
+    with self.cached_session(use_gpu=True) as sess:
+      image = array_ops.reshape(
+          constant_op.constant([], dtypes.uint8), [0, 0, 3])
+      with self.assertRaisesOpError(r"image data should not be empty"):
+        png = image_ops.encode_png(image)
```

Comment:
So, I tested internally and the error doesn't seem to be raised always, that's why the test fails (in 1 / 4 attempts).